[Bindings] Update yaml snippet to match updated qs (#3660)
dapr/docs: daprdocs/content/en/developing-applications/building-blocks/bindings/bindings-overview.md

- Adding or configuring a third-party Twilio SDK
- Worrying about polling from Twilio (or using WebSockets, etc.)

<img src="/images/binding-overview.png" width=1000 alt="Diagram showing bindings">

In the above diagram:

> - The input binding triggers your application using the endpoint name (which matches your input binding component name) and the method. 

Reviewer: ```suggestion
- The input binding triggers a method on your application.
```

Reviewer: Can we change this sentence above "For example, with bindings, your microservice can respond to incoming Twilio/SMS messages without:"
to
"For example, with bindings, your application can respond to incoming Twilio/SMS messages without:"
prefer to use application here
